### PR TITLE
Begin documenting the development environment

### PR DESCRIPTION
**Pull request by ramonh**
This addresses part of #32

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,36 @@
...
+# RubyGems Adoption Center
 [![Build Status](https://travis-ci.org/rubygems/adoption-center.svg?branch=master)](https://travis-ci.org/rubygems/adoption-center)
 [![Code Climate](https://codeclimate.com/github/rubygems/adoption-center/badges/gpa.svg)](https://codeclimate.com/github/rubygems/adoption-center)
 
-Gem adoption center
+## Development
+
+Clone the repository
+
+		git clone git@github.com:rubygems/adoption-center.git
```
Comment (qrush) [inline README.md]: Should only need 4 spaces.

**Comment by ramonh** [comment]
Thank you for the tip, @qrush! Should be fixed now

**Comment by ramonh** [comment]
Is it best to rebase?

**Comment by sebasoga** [comment]
@ramonh if you could squash those commits into 1 it would be great

**Pull request merged**